modules/end-user-guide/partials/proc_overriding-a-secrets-mount-path-for-a-workspace-component.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// mounting-a-secret-as-a-file-or-an-environment-variable-into-a-workspace-container
+
+[id="overriding-a-secrets-mount-path-for-a-workspace-component_{context}"]
+= Overriding a secret's mount path for a workspace component
```

Comment:
This shouldn't be a possessive, but given that it's reflected in the file name, I leave it up to you when to fix it (doesn't need to be now).